src/components/DataSession/OperationWizard.vue

```diff
@@ -158,25 +171,55 @@ function selectImage(inputKey, imageIndex) {
       </v-row>
     </v-card-text>
     <v-slide-y-reverse-transition hide-on-leave>
-      <v-card-text v-show="page == 'configure'" class="wizard-card">
-        <div v-for="(inputDescription, inputKey) in selectedOperationInputs" :key="inputKey"
-          class="operation-input-wrapper">
-          <v-text-field v-if="inputDescription.type !== 'file'" v-model="selectedOperationInput[inputKey]"
-            :label="inputDescription.name" :type="inputDescription.type" class="operation-input" />
-          <div v-else-if="inputDescription.type == 'file'" class="images-container">
-            <image-grid :images="images" :column-span="calculateColumnSpan(images.length, imagesPerRow)"
-              class="wizard-images" :allow-selection="true" @select-image="selectImage(inputKey, $event)" />
+      <v-card-text
+        v-show="page == 'configure'"
+        class="wizard-card"
+      >
+        <div
+          v-for="(inputDescription, inputKey) in selectedOperationInputs"
+          :key="inputKey"
+          class="operation-input-wrapper"
+        >
+          <v-text-field
+            v-if="inputDescription.type !== 'file'"
+            v-model="selectedOperationInput[inputKey]"
+            :label="inputDescription.name"
+            :type="inputDescription.type"
+            class="operation-input"
+          />
+          <div
+            v-else-if="inputDescription.type == 'file'"
+            class="images-container"
+          >
+            <div class="input-images">
+              Input Images
```

Comment:
This label should be pulled from the inputDescription.name (we can have multiple file inputs and don't want them all just saying "Input Images". Also it would be nice to style the label exactly the same as the label on other inputs, i.e. "Duration" in your screenshot.

Reply:
my IDE is her own person and we respect her. Thanks for the feedback! done :)